docs/doxygen/porting.txt

```diff
@@ -8,14 +8,14 @@ A port to a new system must provide the following components:
 3. [Time Function](@ref porting_time)
 
 @section porting_config Configuration Macros
-@brief Settings that must be set as macros in the config header `core_mqtt_config.h`, or passed in as compiler options
+@brief Settings that must be set as macros in the config header `core_mqtt_config.h`, or passed in as compiler options.
 
-@see [Configurations](@ref core_mqtt_config)
+@note If a custom configuration header `core_mqtt_config.h` is not provided, then the @ref MQTT_DO_NOT_USE_CUSTOM_CONFIG macro must be defined.
 
-@note Regardless of whether the following macros are defined in `core_mqtt_config.h` or passed as compiler options,
-the `core_mqtt_config.h` file must be included to build the MQTT library.
+@see [Configurations](@ref core_mqtt_config)
 
-The following macros can be configured for the managed MQTT library:
+The following macros can be configured for the MQTT library:
+ - @ref MQTT_DO_NOT_USE_CUSTOM_CONFIG <br>
```

Comment:
This configuration shouldn't be placed here as it can the incorrect impression that it has to be defined with the `core_mqtt_config.h` file.

Reply:
Then where do you want it?

Reply:
I'll just remove it then. It's already in the `@note` above